app/services/bulk_ingest_service.rb

```diff
@@ -21,6 +21,14 @@ def attach_each_dir(base_directory:, property: nil, file_filter: nil, **attribut
     end
   end
 
+  # Determines whether or not the string encodes a bib. ID
+  # See SourceMetadataIdentifierValidator#validate
+  # @param [String] value
+  # @return [Boolean]
+  def bibdata_identifier?(value)
+    RemoteRecord.retrieve(value).success?
```

Comment:
I believe this will succeed whether it's a bibdata id or a pulfa id.

There are some pulfa resources on our list of things to ingest, so maybe this test is what we want, but just with a different method name like `valid_identifier?` or something?

the `bibdata?` implementation is rather naive
https://github.com/pulibrary/pul_metadata_services/blob/master/lib/pul_metadata_services/client.rb#L19-L21
I guess another option would be to change it there or implement something more specific here.

Reply:
I have created a separate issue for pul_metadata_services: https://github.com/pulibrary/pul_metadata_services/issues/32